docs/scaling.md

```diff
@@ -0,0 +1,43 @@
+# Scaling
```

Comment:
👍

May also be worth putting a note that Argo puts load on the Kubernetes master/etcd so that's a place to look too with regards to scaling.